tests/test_subscriptions.py

```diff
@@ -0,0 +1,113 @@
+from mollie.api.objects.list import List
+from mollie.api.objects.subscription import Subscription
+from mollie.api.objects.customer import Customer
+CUSTOMER_ID = 'cst_kEn1PlbGa'
+SUBSCRIPTION_ID = 'sub_rVKGtNd6s3'
+
+
+def test_customer_subscriptions_all(client, response):
+    """Retrieve a list of subscriptions"""
+    response.get('https://api.mollie.com/v2/customers/%s/subscriptions' % CUSTOMER_ID, 'subscription_all')
+
+    subscriptions = client.customer_subscriptions.with_parent_id(CUSTOMER_ID).all()
+    assert isinstance(subscriptions, List)
+    assert subscriptions.count == 3
+    iterated = 0
+    iterated_subscription_ids = []
+    for subscription in subscriptions:
+        assert isinstance(subscription, Subscription)
+        iterated += 1
+        assert subscription.id is not None
+        iterated_subscription_ids.append(subscription.id)
+    assert iterated == subscriptions.count, 'Unexpected amount of subscriptions retrieved'
+    assert len(set(iterated_subscription_ids)) == subscriptions.count, 'Expected unique subscription ids retrieved'
+
+
+def test_get_customer_subscription_by_id(client, response):
+    """Retrieve a single subscription by ID """
+    response.get('https://api.mollie.com/v2/customers/%s/subscriptions/%s' % (CUSTOMER_ID, SUBSCRIPTION_ID),
+                 'subscription_single')
+
+    subscription = client.customer_subscriptions.with_parent_id(CUSTOMER_ID).get(SUBSCRIPTION_ID)
+    assert subscription.resource == 'subscription'
+    assert subscription.id == SUBSCRIPTION_ID
+    assert subscription.mode == 'live'
+    assert subscription.created_at == '2016-06-01T12:23:34+00:00'
+    assert subscription.is_active() is True
+    assert subscription.is_suspended() is False
+    assert subscription.is_pending() is False
+    assert subscription.is_completed() is False
+    assert subscription.is_canceled() is False
+    assert subscription.amount['value'] == '25.00'
+    assert subscription.amount['currency'] == 'EUR'
+    assert subscription.times == 4
+    assert subscription.interval == '3 months'
+    assert subscription.description == 'Quarterly payment'
+    assert subscription.method == 'ideal'
+    assert subscription.webhook_url == 'https://webshop.example.org/payments/webhook'
+
+
+def test_get_all_subscription_by_customer_object(client, response):
```

Comment:
Move contents of this test to `test_customers`, write new test that looks into `client.customer_subscriptions.on (CUSTOMER).all()` en `client.customer_subscriptions.on (CUSTOMER).get()` (zie test_get_customer_mandate_by_customer`).